app/controllers/shelters_controller.rb

```diff
@@ -14,6 +14,8 @@ def create
 
   def show
     @shelter = Shelter.find(params[:id])
+
+    # @shelter_reviews = ShelterReview.find(params[:id])
```

Comment:
This is just a guess really. I'm not sure this code is right because I didn't get a chance to work with it, but I commented it out as a reminder to come back to it.

Reply:
Thanks for trying this out.  I will check it out this morning and report back on what I find!